variables.tf

```
@@ -2,7 +2,7 @@
 # Common variables
```

[Naming - Blocking] Variable uses mixed case (gRoup) instead of snake_case. This violates Terraform conventions and breaks consistency.

```suggestion
variable "resource_group_id" {
```